src/main.ts

```diff
@@ -29,6 +30,15 @@ async function run(): Promise<void> {
         }
         await sonarScanner.run(inputs.sonarScannerVersion, sonarArgs);
         break;
+      case 'maven':
+        if (!inputs.sonarScannerVersion) {
+          throw new Error(
+            `sonar-scanner-version is required when using sonar-scanner`
+          );
+        }
```

Comment:
This is probably not required, because it will use the version that maven installs.